### PR TITLE
Implemented Portal module. Added styling

### PR DESCRIPTION
**Pull request by angushtlam**

### CODE REVIEW

File: swe0/ext/portal/views.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import os
```
Comment (StephenOrJames) [inline swe0/ext/portal/views.py]: Unused import?

File: swe0/ext/portal/views.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+from flask import render_template
+from flask_login import current_user, login_required
```
Comment (StephenOrJames) [inline swe0/ext/portal/views.py]: Unused imports?

Comment (angushtlam) [reply swe0/ext/portal/views.py]: Removed them

**Comment by angushtlam** [comment]
imma self-lgtm cuz i removed the unused imports

**Pull request merged**